site/content/docs/manifest/pipeline.en.md

```diff
@@ -17,7 +17,7 @@ List of all available properties for a Copilot pipeline manifest. To learn more
             # connection_name: a-connection
     
         build:
-          image: aws/codebuild/amazonlinux2-x86_64-standard:4.0
+          image: aws/codebuild/amazonlinux2-x86_64-standard:5.0
```

Comment:
Do you know if the following scenarios will break for pipeline users

1. They don't update the pipeline by running `copilot pipeline deploy` again so the codebuild won't be updated. However, they updated the buildspec to use the latest Copilot.
2. They updated the pipeline and the codebuild was upgrade to `amazonlinux2-x86_64-standard:5.0`. However, they forgot to update the buildspec file.

If so, we might want to make sure they are fully aware of the reason why this happen and what to do to get it around (by looking at the codebuild log).

Reply:
I did not test these scenarios. I will  deploy a pipeline to test these

Reply:
I tested both the scenarios, surprisingly it did not error out.

Reply:
Cool! That's even better. It seems like we don't depend on the current env nodejs when they use Copilot.

Reply:
I have only default buildspec that copilot generates other than changing it to latest version.

what if the customer have some commands like `make test` whey they are running some unit tests for their application based on nodejs environment.

Do we need consider this scenario?

Reply:
Hmm it should be good i think, as they should be aware of their dependency on nodejs version. Whereas if the default scenario breaks, they probably don't know why nodejs version would even matter.